source/wp-content/themes/wporg-news-2021/block-templates/404.html

```diff
@@ -1,24 +1,25 @@
 <!-- wp:template-part {"slug":"header","align":"full","className":"site-header-container"} /-->
 
-<!-- wp:group {"layout":{"inherit":true},"tagName":"main"} -->
+<!-- wp:group {"tagName":"main","className":"site-content-container"} -->
 <main class="wp-block-group site-content-container">
+	<p class="error404__oops" aria-hidden="true">
+		Oops!
+	</p>
 
-<!-- wp:heading {"level":1,"fontSize":"large"} -->
-<h1 class="has-large-font-size">This page doesn't exist.</h1>
-<!-- /wp:heading -->
+	<!-- wp:heading {"level":1} -->
+	<h1>This page doesn't exist.</h1>
+	<!-- /wp:heading -->
 
-<!-- wp:html -->
-<div>
-	Go to
-	<ul class="is-inline"><!-- wp:home-link {"label":"the homepage","tagName":"div"} /--></ul>,
-	or try searching from the field below.
-</div>
-<!-- /wp:html -->
+	<!-- wp:html -->
+	<div>
+		Go to
+		<ul class="is-inline"><!-- wp:home-link {"label":"the homepage","tagName":"div"} /--></ul>,
```

Comment:
🤔  it would be nice to simplify it, but I can't think of a HTML-only URL that would be correct for local, staging, and prod.

`<a href="/news/">` would be a 404 on local, and the prod site on staging.

xref https://github.com/WordPress/gutenberg/issues/35926

We could write a custom block if you think the awkward semantics are bad enough to justify it.

Reply:
Well, maybe having the URLs wrong on local/staging is a smaller problem than the alternatives. That'd work for me.

Reply:
changed in https://github.com/WordPress/wporg-news-2021/pull/314/commits/7bd6a80b0eecf72ae3d9bf0e30184f9efa8cc216